src/components/NormalLeftRow/index.jsx

```diff
@@ -12,36 +18,59 @@ function NormalLeftRow({ schema, classes }) {
     object: '{',
     string: '"..."',
   }[schema.type];
-  /** Create blank line paddings only for additional keywords that
-   *  will have their own lines on the according right row.
-   *  This enables the left row to have matching number of lines with
-   *  the right row and align the lines and heights between the two rows.
+  /**
+   * Create blank line paddings only for additional keywords that
+   * will have their own lines on the according right row.
+   * This enables the left row to have matching number of lines with
+   * the right row and align the lines and heights between the two rows.
    */
-  const nonPaddedKeywords = ['type', 'description', 'name'];
+  const nonPaddedKeywords = [
+    'type',
+    'description',
+    'name',
+    'items',
+    'contains',
+  ];
   const blankLinePaddings = [];
 
   Object.keys(schema).forEach(keyword => {
     if (!nonPaddedKeywords.includes(keyword)) {
-      blankLinePaddings.push(<br key={keyword} className={classes.line} />);
+      blankLinePaddings.push(
+        <Typography key={`${keyword} line`} className={classes.line}>
+          <br />
+        </Typography>
+      );
     }
   });
 
   return (
-    <div className={classes.row}>
-      <p className={classes.line}>
-        {name}
+    <div key={schema.type} className={classes.row}>
+      <Typography component="div" className={classes.line}>
```

Comment:
The text is rather large. Maybe we can try something like `variant="subtitle2"`.

Reply:
Applied changes!